docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: "3.3"
```

Comment:
>  `version` is obsolete

Reply:
[Couldn't find exactly but many threads say it](https://stackoverflow.com/questions/76156527/what-does-the-first-line-in-the-docker-compose-yml-file-that-specifies-the-ve)

> As of this writing in May 2023, Docker is planning to desupport the Python version of Compose by the end of June 2023, which will reduce the number of options in this matrix. In particular, this will mean the version: line is ignored always